src/app/db/tables.py

```diff
@@ -3,162 +3,24 @@
-    Column("login", String(50), unique=True),
-    Column("owner_id", Integer, ForeignKey("users.id")),
-    Column("access_id", Integer, ForeignKey("accesses.id", ondelete="CASCADE"), unique=True),
-    Column("specs", String(50)),
-    Column("software_hash", String(16), default=None, nullable=True),
-    Column("angle_of_view", Float(2, asdecimal=True)),
-    Column("elevation", Float(1, asdecimal=True), default=None, nullable=True),
-    Column("lat", Float(4, asdecimal=True), default=None, nullable=True),
-    Column("lon", Float(4, asdecimal=True), default=None, nullable=True),
-    Column("yaw", Float(1, asdecimal=True), default=None, nullable=True),
-    Column("pitch", Float(1, asdecimal=True), default=None, nullable=True),
-    Column("last_ping", DateTime, default=None, nullable=True),
-    Column("created_at", DateTime, default=func.now())
-)
-
-
-class MediaType(str, enum.Enum):
-    image: str = 'image'
-    video: str = 'video'
-
-
-media = Table(
-    "media",
-    metadata,
-    Column("id", Integer, primary_key=True),
-    Column("device_id", Integer, ForeignKey("devices.id")),
-    Column("bucket_key", String(100), nullable=True),
-    Column("type", Enum(MediaType), default=MediaType.image),
-    Column("created_at", DateTime, default=func.now()),
-)
-
-
-installations = Table(
-    "installations",
-    metadata,
-    Column("id", Integer, primary_key=True),
-    Column("device_id", Integer, ForeignKey("devices.id")),
-    Column("site_id", Integer, ForeignKey("sites.id")),
-    Column("start_ts", DateTime, nullable=False),
-    Column("end_ts", DateTime, default=None, nullable=True),
-    Column("is_trustworthy", Boolean, default=True),
-    Column("created_at", DateTime, default=func.now()),
-)
-
-
-alerts = Table(
-    "alerts",
-    metadata,
-    Column("id", Integer, primary_key=True),
-    Column("device_id", Integer, ForeignKey("devices.id")),
-    Column("event_id", Integer, ForeignKey("events.id")),
-    Column("media_id", Integer, ForeignKey("media.id"), default=None),
-    Column("azimuth", Float(4, asdecimal=True), default=None),
-    Column("lat", Float(4, asdecimal=True)),
-    Column("lon", Float(4, asdecimal=True)),
-    Column("is_acknowledged", Boolean, default=False),
-    Column("created_at", DateTime, default=func.now()),
-)
-
-
-webhooks = Table(
-    "webhooks",
-    metadata,
-    Column("id", Integer, primary_key=True),
-    Column("callback", String(50), nullable=False),
-    Column("url", String(100), nullable=False),
-)
+           'users', 'accesses', 'groups', 'sites', 'events', 'devices', 'media',
+           'installations', 'alerts', 'webhooks']
+
+users = Users.__table__
+accesses = Accesses.__table__
+groups = Groups.__table__
+sites = Sites.__table__
+events = Events.__table__
+devices = Devices.__table__
+media = Media.__table__
+installations = Installations.__table__
+alerts = Alerts.__table__
+webhooks = Webhooks.__table__
+
+metadata = Base.metadata
```

Comment:
Is this remaining only for the transition or will we have to keep it?

Reply:
It remains to keep the legacy structure of the code and CRUDs. Maybe we will be able to migrate in a latter PR but that will be a heavy one. 
For now, I would prefer to just leverage the ORM capacities to ease some implementation  (e.g the fetch_all with the same group_id).

Reply:
Makes sense, thanks!